substrate/frame/staking/src/lib.rs

```diff
@@ -1223,3 +1223,67 @@ impl BenchmarkingConfig for TestBenchmarkingConfig {
 	type MaxValidators = frame_support::traits::ConstU32<100>;
 	type MaxNominators = frame_support::traits::ConstU32<100>;
 }
+
+/// Controls validator disabling
+pub trait DisablingStrategy<T: Config> {
+	/// Make a decision if an offender should be disabled or not. The result is a `Vec` of validator
+	/// indices that should be disabled
+	fn make_disabling_decision(
```

Comment:
nitpick: since this trait is already called `DisablingStrategy`, the method could be simplified as `decision` or something along those lines so that the naming doesn't stutter.